test/core/extensions/io_extension_test.cpp

```diff
@@ -76,8 +93,8 @@ TEST_F(IOExtensionTest, PrintNum) {
 TEST_F(IOExtensionTest, PrintUTF8) {
   WasmPointer data = 0;
   WasmSize size = utf8_bytes_.size();
-  Buffer buf(utf8_bytes_);
+  std::string buf(&utf8_bytes_.front(), &utf8_bytes_.back());
```

Comment:
Probably is more pure to use iterators instead raw pointers: `utf8_bytes_.begin()` and `utf8_bytes_.end()`

Reply:
Maybe you are wright, for test it can be good enough. FYI: string construction with 2 pointers much faster, because of single allocation of ```size = ptr_end - ptr_begin + 1;```. I'm not sure that ```distance``` will be used for iterators. It should be checked additionaly.

Reply:
Inside iterator is light-weight wrapper over pointer. And this is RandomAccessIterator of course.